tests/system/python/e2e/test_e2e_filter_fft_threshold.py

```diff
@@ -145,6 +157,30 @@ def test_end_to_end(self, start_south_north, disable_schedule, foglamp_url, read
         if not skip_verify_north_interface:
             self._verify_egress(read_data_from_pi, pi_host, pi_admin, pi_passwd, pi_db, wait_time, retries)
 
+        tracking_details = self.get_asset_tracking_details(foglamp_url, "Ingest")
+        assert len(tracking_details["track"]), "Failed to track Ingest event"
+        tracked_item = tracking_details["track"][0]
+        assert "playfilter" == tracked_item["service"]
+        assert "e2e_fft_threshold FFT" == tracked_item["asset"]
+        assert "playback" == tracked_item["plugin"]
+
+        # TODO: Add asset tracker entry for fft and threshold filters
```

Comment:
TODO: FOGL-xxx ? ....

Reply:
Issue was not created then ,thats why I haven't put it.